web-client/src/views/Practitioners/PractitionerInformation.tsx

```diff
@@ -11,17 +14,63 @@ import React from 'react';
 
 export const PractitionerInformation = connect(
   {
+    gotoPrintPractitionerCasesSequence:
+      sequences.gotoPrintPractitionerCasesSequence,
     onPractitionerInformationTabSelectSequence:
       sequences.onPractitionerInformationTabSelectSequence,
-    showDocumentationTab:
-      state.practitionerInformationHelper.showDocumentationTab,
+    practitionerDetailHelper: state.practitionerDetailHelper,
+    practitionerInformationHelper: state.practitionerInformationHelper,
+    setPractitionerClosedCasesPageSequence:
+      sequences.setPractitionerClosedCasesPageSequence,
+    setPractitionerOpenCasesPageSequence:
+      sequences.setPractitionerOpenCasesPageSequence,
     showModal: state.modal.showModal,
   },
   function PractitionerInformation({
+    gotoPrintPractitionerCasesSequence,
     onPractitionerInformationTabSelectSequence,
-    showDocumentationTab,
+    practitionerInformationHelper,
+    setPractitionerClosedCasesPageSequence,
+    setPractitionerOpenCasesPageSequence,
     showModal,
   }) {
+    const numOpenCases = practitionerInformationHelper.openCasesTotal || 0;
+    const numClosedCases = practitionerInformationHelper.closedCasesTotal || 0;
+
+    const openPagesPaginator = () => {
+      return (
+        practitionerInformationHelper.showOpenCasesPagination && (
+          <Paginator
+            currentPageIndex={practitionerInformationHelper.openCasesPageNumber}
+            totalPages={practitionerInformationHelper.totalOpenCasesPages}
+            onPageChange={selectedPage => {
+              setPractitionerOpenCasesPageSequence({
+                pageNumber: selectedPage,
+              });
+            }}
+          />
+        )
+      );
+    };
+
+    const closedPagesPaginator = () => {
+      return (
+        practitionerInformationHelper.showClosedCasesPagination && (
+          <Paginator
+            currentPageIndex={
+              practitionerInformationHelper.closedCasesPageNumber
+            }
+            totalPages={practitionerInformationHelper.totalClosedCasesPages}
+            onPageChange={selectedPage => {
+              setPractitionerClosedCasesPageSequence({
+                pageNumber: selectedPage,
+              });
+            }}
+          />
+        )
+      );
+    };
```

Comment:
Ideally, I would want pagination to live on `PractitionerCaseList` directly and be handled by using `useState` and URL params; however, this goes against our current use of Cerebral.

I chose what I thought was the next best option: Rather than pass in specific global state into `PractitionerCaseList`, which makes the component "messier," I chose to pass in the list of cases into `PractitionerCaseList` and handle pagination here in this component since it is already connected to global state.